components/form-builder/app/Preview.tsx

```diff
@@ -26,14 +26,19 @@ export const Preview = () => {
 
   const router = useRouter();
   const { t: t1 } = useTranslation();
-  const { t } = useTranslation("form-builder");
+  const { t, i18n } = useTranslation("form-builder");
   const language = translationLanguagePriority;
   const currentForm = getRenderedForm(formRecord, language, t);
 
   return (
     <>
       <span className="bg-purple-200 p-2 inline-block mb-1">{t("page1")}</span>
-      <div className="border-3 border-dashed border-blue-focus p-4 mb-8 pointer-events-none">
+      <div
+        className="border-3 border-dashed border-blue-focus p-4 mb-8 pointer-events-none"
+        {...(i18n.language !== translationLanguagePriority && {
+          lang: translationLanguagePriority,
+        })}
```

Comment:
Would be cleaner to put the lang attribute on each form element vs. higher up in a parent element and then overriding with buttons/text not matching. The preview and translate was somewhat complex and this seemed better since it resulted in touching less code